portage-stubs/__init__.pyi

```diff
@@ -17,7 +18,8 @@ class dbapi:
 
     def xmatch(self, level: Literal['bestmatch-visible', 'match-all-cpv-only', 'match-all',
                                     'match-visible', 'minimum-all', 'minimum-visible',
-                                    'minimum-all-ignore-profile'], origdep: str) -> list[str] | str:
+                                    'minimum-all-ignore-profile'],
+                origdep: str) -> Union[Sequence[str], str]:
```

Comment:
```suggestion
                origdep: str) -> list[str] | str:
```

Does it return a list of strings or just `str` or either? You should make it return the real type. Input parameters can be things like `Sequence`.

Also note that `Sequence[str] == str` for reasons.